packages/manager/modules/pci/src/projects/project/failover-ips/imports/imports.html

```diff
@@ -0,0 +1,20 @@
+<div data-ui-view>
```

Comment:
Missing some `data-` prefixes (`heading` and `property`).

Reply:
Custom UI-Kit element -> not required ?

Reply:
The`data-` prefix is already spread into a large amount of the OVH UI-Kit implementation.